src/util/system.cpp

```diff
@@ -367,9 +367,10 @@ bool ArgsManager::ParseParameters(int argc, const char* const argv[], std::strin
 
     // we do not allow -includeconf from command line
```

Comment:
In commit "util: Properly handle -noincludeconf on command line" (fa2fc70adefda51913b714993931e281a67de5e0)

Since it's not obvious how the -noincludeconf case is relevant to this loop (and it already caused a bug) it would be good to mention it in comments. Maybe:

```diff
--- a/src/util/system.cpp
+++ b/src/util/system.cpp
@@ -365,8 +365,9 @@ bool ArgsManager::ParseParameters(int argc, const char* const argv[], std::strin
         m_settings.command_line_options[key].push_back(value);
     }
 
-    // we do not allow -includeconf from command line
+    // we do not allow -includeconf from command line, only -noincludeconf
     if (auto* includes = util::FindKey(m_settings.command_line_options, "includeconf")) {
+        // Range will be empty in -noincludeconf case
         for (const auto& include : util::SettingsSpan(*includes)) {
             error = "-includeconf cannot be used from commandline; -includeconf=" + include.write();
             return false; // pick first value as example
```

```

```

Reply:
Thanks, fixed